variables.tf

```diff
@@ -22,21 +22,25 @@
 # Configure the IBM Cloud provider
 ################################################################
 variable "ibmcloud_api_key" {
+    type        = string
     description = "IBM Cloud API key associated with user's identity"
     default = "<key>"
 }
 
 variable "service_instance_id" {
+    type        = string
     description = "The cloud instance ID of your account"
     default = ""
 }
 
 variable "ibmcloud_region" {
```

Comment:
Need validation as this is a required variable.

Reply:
Added validation to check empty string and default value present in var.tfvars.